index.js

```diff
@@ -5,21 +5,21 @@ var objectAssign = require('object-assign');
 var escapeStringRegexp = require('escape-string-regexp');
 require('string.prototype.repeat');
 
-var CSS_ESCAPED_SPACE = '\\20';
+var CSS_ESCAPED_TAB = '\\9';
 
-// Plugin that adds `:not(#\\20)` selectors to the front of the rule thus increasing specificity
+// Plugin that adds `:not(#\\9)` selectors to the front of the rule thus increasing specificity
 module.exports = postcss.plugin('postcss-increase-specifity', function(options) {
 	var defaults = {
-		// The number of times `:not(#\\20)` is appended in front of the selector
+		// The number of times `:not(#\\9)` is appended in front of the selector
 		repeat: 3,
 		// Whether to add !important to declarations in rules with id selectors
 		overrideIds: true,
 		// The thing we repeat over and over to make up the piece that increases specificity
-		// > Consider use :not(#\20), :not(.\20) and :not(\20)
-		// > Rationale: \20 is a css escape for U+0020 Space, and neither classname, nor id, nor tagname can contain a space
+		// > Consider use :not(#\9), :not(.\9) and :not(\9)
+		// > Rationale: \9 is a css escape for U+0009 Character Tabulation, and neither classname, nor id, nor tagname can contain a tab character
 		// >
-		// > — https://twitter.com/subzey/status/829050478721896448
-		stackableRoot: ':not(#' + CSS_ESCAPED_SPACE + ')'
+		// > — https://twitter.com/subzey/status/829050478721896448 (originally \20)
```

Comment:
@valtlai Let's remove this Twitter quotation now that things have changed and don't match up as nicely.